src/types/commands.ts

```diff
@@ -464,10 +466,14 @@ export interface UpdateFilterCommand extends PositionDependentCommand {
 export interface SetFormattingCommand extends TargetDependentCommand {
   type: "SET_FORMATTING";
   style?: Style;
-  border?: BorderCommand;
```

Comment:
this will also require a migration if we want to avoid the information to be lost when replaying the commands.